### PR TITLE
Fix openvino lin debian ci

### PR DESCRIPTION
**Pull request by praasz**
### Details:
 - Fix Debian Linux CI by providing correct environment variables for python imports and frontend manager.
 - Remove not required paths in Linux CI
 - Add comments in python API 2.0 tests step  which explains environment variables usage.

### Tickets:
 - 92966

### CODE REVIEW

File: .ci/azure/linux_debian.yml [hunk 1]
```diff
@@ -242,9 +243,12 @@ jobs:
       export DATA_PATH=$(MODELS_PATH)
       export MODELS_PATH=$(MODELS_PATH)
       # for libopenvino_template_extension.so
-      export LD_LIBRARY_PATH=$(INSTALL_TEST_DIR):$LD_LIBRARY_PATH
-      export PYTHONPATH=$(INSTALL_TEST_DIR):$(INSTALL_DIR)/python/python3.8:$PYTHONPATH
-      export OV_FRONTEND_PATH=$(INSTALL_TEST_DIR)
+      export LD_LIBRARY_PATH=$(INSTALL_TEST_DIR):$(INSTALL_DIR)/python/python3.8:$LD_LIBRARY_PATH
+      export OV_FRONTEND_PATH=$(PYTHON_WHEEL_INSTALL_DIR)/openvino/libs:$(INSTALL_TEST_DIR)
+      export PYTHONPATH=$(INSTALL_TEST_DIR)
```
Comment (ilya-lavrenov) [inline .ci/azure/linux_debian.yml:248]: it would be nice to add comments what paths are needed for what

Comment (praasz) [reply .ci/azure/linux_debian.yml:248]: Yes, when figure out what has to be set to run these python mock tests then will add comments

**Pull request merged**